components/notifications/emails/factory.js

```diff
@@ -9,7 +9,7 @@ const TYPES                         = require('./enums/types');
 
 const NotificationEmailDefinition = {
 
-    name: 'NotificationsEmails',
+    name: 'NotificationsEmail',
```

Comment:
It should be with "s". No good reason for it, just a convention that all factories names are in plural. `NotificationEmailDefinition` - this also may be changed.